infra-data-lake/localhost/docker-compose.yml

```diff
@@ -48,7 +48,6 @@ services:
     ports:
       - '8890:8888'
     volumes:
-      - ./../../notebook-data-lake/data:/data
```

Comment:
this causes the issue, adding this line back everything works fine. shouldn't be this way.